fuzz/cycle.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Continuosly cycle over fuzz targets running each for 1 hour.
+# It uses high niceness so that other process takes priority.
+# For batch processes niceness works ok, but if you need 
+# interactivity on the server it's not great.
+# In the latter case considering limiting CPU cores dedicated to 
+# fuzzing or change schedules with `chrt`.
+#
+# For hfuzz options see https://github.com/google/honggfuzz/blob/master/docs/USAGE.md
+
+export HFUZZ_BUILD_ARGS='--features honggfuzz_fuzz'
+export HFUZZ_RUN_ARGS='--run_time 3600'
+export RUSTFLAGS='--cfg=rust_secp_fuzz'
```

Comment:
(I know this was copied from my own code, and maybe it's in travis-fuzz.) This line is currently obsolete -- rust-secp doesn't recognize this `cfg` flag and it will always use the mock library when honggfuzz is running.

Having said that, we will probably bring the `cfg` option back soon to deal with https://github.com/rust-bitcoin/rust-secp256k1/issues/486 and others..

Reply:
I removed it, for now...